cmd/manager/main.go

```diff
@@ -128,8 +135,27 @@ func main() {
 		os.Exit(1)
 	}
 
+	disableHTTP2 := func(config *tls.Config) {
```

Comment:
please leave a comment in the code why it is disabled by default

Reply:
Done